versionedRouter.js

```diff
@@ -307,32 +307,54 @@ async function handleValidation(ctx) {
 }
 
 async function routerHandleDest(ctx) {
-  const { destType, input } = ctx.request.body;
-  const routerDestHandler = getDestHandler("v0", destType);
-  if (!routerDestHandler || !routerDestHandler.processRouterDest) {
-    ctx.status = 404;
-    ctx.body = `${destType} doesn't support router transform`;
-    return null;
-  }
   const respEvents = [];
-  const allDestEvents = _.groupBy(input, event => event.destination.ID);
-  await Promise.all(
-    Object.entries(allDestEvents).map(async ([destID, desInput]) => {
-      desInput = processDynamicConfig(desInput, "router");
-      const listOutput = await routerDestHandler.processRouterDest(desInput);
-      respEvents.push(...listOutput);
-    })
-  );
-  respEvents
-    .filter(
-      resp =>
-        "error" in resp &&
-        _.isObject(resp.statTags) &&
-        !_.isEmpty(resp.statTags)
-    )
-    .forEach(resp => {
-      set(resp, "statTags.errorAt", TRANSFORMER_METRIC.ERROR_AT.RT);
-    });
+  let destType;
+  try {
+    const { input } = ctx.request.body;
+    destType = ctx.request.body.destType;
+    const routerDestHandler = getDestHandler("v0", destType);
+    if (!routerDestHandler || !routerDestHandler.processRouterDest) {
+      ctx.status = 404;
+      ctx.body = `${destType} doesn't support router transform`;
+      return null;
+    }
+    const allDestEvents = _.groupBy(input, event => event.destination.ID);
+    await Promise.all(
+      Object.entries(allDestEvents).map(async ([destID, desInput]) => {
+        desInput = processDynamicConfig(desInput, "router");
+        const listOutput = await routerDestHandler.processRouterDest(desInput);
```

Comment:
RT workflow execution is not included here:
I have done that in this: https://github.com/rudderlabs/rudder-transformer/pull/1471

Reply:
Addressed.